dll/cpl/sysdm/lang/es-ES.rc

```diff
@@ -290,7 +290,24 @@ BEGIN
     LTEXT "&Valor:", IDC_STATIC, 7, 32, 68, 8
     EDITTEXT IDC_VARIABLE_VALUE, 75, 30, 145, 12, ES_AUTOHSCROLL
     DEFPUSHBUTTON "Aceptar", IDOK, 116, 50, 50, 14
-    PUSHBUTTON "Cancelar", IDCANCEL, 170, 50, 50, 14
+    PUSHBUTTON "", IDCANCEL, 170, 50, 50, 14
```

Comment:
Why did you remove the cancel text here?

Reply:
By mistake.